ansible_collections/arista/avd/roles/eos_l3ls_evpn/README.md

```diff
@@ -0,0 +1,1164 @@
+    l2vlans:
+      < 1-4096 >:
+        vni_override: < 1-16777215 >
+        name: < description >
+        tags: [ < tag_1 >, < tag_2 > ]
+      < 1-4096 >:
+        name: < description >
+        tags: [ < tag_1 >, < tag_2 > ]
+
+```
+
+**Example:**
+
+```yaml
+# mlag_ibgp_peering_vrfs:
+#   base_vlan: 3000
+
+tenants:
+  Tenant_A:
+    mac_vrf_vni_base: 10000
+    vrfs:
+      Tenant_A_OP_Zone:
+        vrf_vni: 10
+        vtep_diagnostic:
+          loopback: 100
+          loopback_ip_range: 10.255.1.0/24
+        svis:
+          110:
+            name: Tenant_A_OP_Zone_1
+            tags: [ opzone ]
+            enabled: true
+            ip_subnet: 10.1.10.0/24
+          111:
+            vni_override: 50111
+            name: Tenant_A_OP_Zone_2
+            tags: [ opzone ]
+            enabled: true
+            ip_subnet: 10.1.11.0/24
+      Tenant_A_WEB_Zone:
+        vrf_vni: 11
+        svis:
+          120:
+            name: Tenant_A_WEB_Zone_1
+            tags: [ web, erp1 ]
+            enabled: true
+            ip_subnet: 10.1.20.0/24
+          121:
+            name: Tenant_A_WEBZone_2
+            tags: [ web ]
+            enabled: true
+            ip_subnet: 10.1.21.0/24
+      Tenant_A_APP_Zone:
+        vrf_vni: 12
+        svis:
+          130:
+            name: Tenant_A_APP_Zone_1
+            tags: [ app, erp1 ]
+            enabled: true
+            ip_subnet: 10.1.30.0/24
+          131:
+            name: Tenant_A_APP_Zone_2
+            tags: [ app ]
+            enabled: true
+            ip_subnet: 10.1.31.0/24
+      Tenant_A_DB_Zone:
+        vrf_vni: 13
+        svis:
+          140:
+            name: Tenant_A_DB_BZone_1
+            tags: [ db, erp1 ]
+            enabled: true
+            ip_subnet: 10.1.40.0/24
+          141:
+            name: Tenant_A_DB_Zone_2
+            tags: [ db ]
+            enabled: true
+            ip_subnet: 10.1.41.0/24
+      Tenant_A_WAN_Zone:
+        vrf_vni: 14
+        svis:
+          150:
+            name: Tenant_A_WAN_Zone_1
+            tags: [ wan ]
+            enabled: true
+            ip_subnet: 10.1.40.0/24
+    l2vlans:
+      160:
+        vni_override: 55160
+        name: Tenant_A_VMOTION
+        tags: [ vmotion ]
+      161:
+        name: Tenant_A_NFS
+        tags: [ nfs]
+
+  Tenant_B:
+    mac_vrf_vni_base: 20000
+    vrfs:
+      Tenant_B_OP_Zone:
+        vrf_vni: 20
+        svis:
+          210:
+            name: Tenant_B_OP_Zone_1
+            tags: [ opzone ]
+            enabled: true
+            ip_subnet: 10.2.10.0/24
+          211:
+            name: Tenant_B_OP_Zone_2
+            tags: [ opzone ]
+            enabled: true
+            ip_subnet: 10.2.11.0/24
+      Tenant_B_WAN_Zone:
+        vrf_vni: 21
+        svis:
+          250:
+            name: Tenant_B_WAN_Zone_1
+            tags: [ wan ]
+            enabled: true
+            ip_subnet: 10.2.50.0/24
+```
+
+### Server Edge Port Connectivity
+
+- The
```

Comment:
Description is missing

Reply:
updated with commit ac4c80c6d81942a24db83c4b209ce7292aaf1b27